geozero/src/geojson/geojson_reader.rs

```diff
@@ -96,7 +96,7 @@ fn process_geojson<P: FeatureProcessor>(gj: &GeoGeoJson, processor: &mut P) -> R
                 }
                 if let Some(ref geometry) = feature.geometry {
                     processor.geometry_begin()?;
-                    process_geojson_geom_n(geometry, idx, processor)?;
+                    process_geojson_geom_n(geometry, 0, processor)?;
```

Comment:
Note this change - It might be controversial!

Without it, the second line of the csv will have it's geometry prefixed with an unwanted ",".

I'm willing to be talked out of this if you have a good rationale.

Alternatively, I could revert this and hack around this in the CSVWriter, but I think it's arguably correct for the first geometry in each feature to have index 0.